### PR TITLE
chore: inject css to components

### PR DESCRIPTION
**Pull request by alexarassat**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+
+function getEntries(dir) {
+  return Object.assign.apply({},
+    fs.readdirSync(dir, {
+      withFileTypes: true
+    })
+    .filter(dirent => dirent.isDirectory())
```
Comment (ondreian) [inline webpack.config.js:9]: dirent is a really weird variable name, what's this mean?

**Review by ondreian — COMMENTED** [review]
seems reasonable!

File: webpack.config.js (hunk 1 above)
Comment (alexarassat) [reply webpack.config.js:9]: https://nodejs.org/api/fs.html#class-fsdirent

**Review by ondreian — APPROVED** [review]
lgtm

**Pull request merged**